services/121-service/src/seed-data/program/program-nlrc-ocw.json

```diff
@@ -42,6 +42,10 @@
         {
           "name": "coverLetterCode",
           "value": "INTERSOLVE_VISA_COVERLETTER_CODE"
+        },
+        {
+          "name": "fullName",
+          "value": "lastName"
```

Comment:
@RubenGeo We thought of this as a 'mapping' configuration. For example, when a different program wants to use visa and has the full name in a column called 'completeName' then this would 'map' that column to the correct one